examples/src/index.tsx

```diff
@@ -3,6 +3,9 @@ import React from "react";
 
 import Viewer from './Viewer';
 
+declare const SIMULARIUM_USE_OCTOPUS: boolean;
+declare const SIMULARIUM_USE_LOCAL_BACKEND: boolean;
```

Comment:
Is this a way to tell the linters etc that these vars will be injected by external means?

Reply:
yes